### DISCUSSION TITLE
Affected --plain

### DISCUSSION THREAD

**Discussion by damianroszyk** [nrwl/nx#7467] [General]
Hi,

is it possible to use `--plain` with `nx affected` command ?

Example: 

` nx affected --target build-storybook --plain` 

I'd like to know if there are any changes to publish with storybook, but it looks like --plain is skipped here.

**Comment by brandonroberts** [comment]
You can use `nx affected:libs --target build-storybook --plain` to get a plain delimited list

**Reply by damianroszyk** [reply]
Hey @brandonroberts thanks for your reply!

Ok I think I have misunderstood the command because now I get listed all of my libs affected, not only the ones with target `build-storybook`.
Do you know how can I achieve that: get all affected apps/libs names with target `build-storybook` so I can run `chromatic` against ?
